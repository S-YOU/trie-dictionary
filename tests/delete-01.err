nodes: 2
nodes: 4
nodes: 5
nodes: 6
nodes: 5
nodes: 6
nodes: 5
nodes: 3
