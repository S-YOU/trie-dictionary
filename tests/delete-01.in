insert aaa
insert aab
insert aaba
insert aac
delete 1
find aab
insert a
find aab
delete 1
delete 0
find aab
delete 2
find aab
