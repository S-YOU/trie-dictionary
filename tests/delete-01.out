word number: 0
word number: 1
word number: 2
word number: 3
deleted: 1
YES
word number: 4
YES
ignored
deleted: 0
YES
deleted: 2
NO
